### ISSUE TITLE
[joomla.org] 

### ISSUE THREAD

**Issue by markcjf** [joomla/joomla-websites#1902]
When selecting learn from any of your links the following show....

MediaWiki internal error.

Original exception: [f4509edbccdf53b1776d74c7] /Special:MyLanguage/Tutorials:Beginners Error from line 69 of /home/docsj/public_html/includes/skins/SkinTemplate.php: Cannot instantiate abstract class QuickTemplate
Backtrace:

**Comment by pe7er** [comment]
I confirm this issue, e.g. https://docs.joomla.org/Retrieving_request_data_using_JInput

![screencapture-docs-joomla-org-Retrieving-request-data-using-JInput-2023-06-09-16_11_05](https://github.com/joomla/joomla-websites/assets/1217850/2ee4be23-d814-4708-aa89-e2b59351a59e)

**Comment by pe7er** [comment]
I checked, and from my location (NL) it's still giving the same error as I mentioned above, though the HTTP status of the page is 200...

**Comment by webmasterab** [comment]
I also view it from the Netherlands and just see the page

**Comment by ceford** [comment]
If I use this link in Firefox I get the expected page:

https://docs.joomla.org/Special:MyLanguage/Planning_for_Mini-Migration_-_Joomla_3.10.x_to_4.x

But if I use wget in a terminal I get the stack trace shown by @pe7er

wget https://docs.joomla.org/Special:MyLanguage/Planning_for_Mini-Migration_-_Joomla_3.10.x_to_4.x

There has been a Forum discussion on this problem:

https://forum.joomla.org/viewtopic.php?f=303&t=1002121

Baffles me!

**Comment by wilsonge** [comment]
So I definitely introduced this on Monday when I was trying to update the template to follow some of the newer Mediawiki 1.35 best practices in anticipation of doing a 1.39 upgrade in the not so distant future. Why it seems to have been cached I have no clue. I've just purged all cache within the mediawiki. Then all the lscache and no joy

Then when all else failed I went to go ponder things and remove the setting to show the stack trace from the config because I'd not meant to leave that enabled. And on removing that setting suddenly the wget thing worked (at least for me) too. So I'm totally baffled :D

But hopefully maybe I've fixed it without understanding who/what/where and why
